docs-app/public/docs/3-components/accordion.md

```diff
@@ -0,0 +1,291 @@
+# Accordion
+
+An accordion component is an element that organizes content into collapsible sections, enabling users to expand or collapse them for efficient information presentation and navigation.
+
+`<Accordion::Accordion />` can be used in any design system.
+
+## Examples
+
+<details open>
+<summary><h3>Simple Accordion</h3></summary>
+
+```gjs live preview
+import { Accordion } from 'ember-primitives';
+
+<template>
+  <Accordion @type="single" as |A|>
+    <A.Item class="accordion-item" @value="what" as |I|>
+      <I.Header as |H|>
+        <H.Trigger class="accordion-trigger">What is Ember?</H.Trigger>
+      </I.Header>
+      <I.Content class="accordion-content">Ember.js is a productive, battle-tested JavaScript framework for building modern web applications. It includes everything you need to build rich UIs that work on any device.</I.Content>
+    </A.Item>
+    <A.Item class="accordion-item" @value="why" as |I|>
+      <I.Header as |H|>
+        <H.Trigger class="accordion-trigger">Why should I use Ember?</H.Trigger>
+      </I.Header>
+      <I.Content class="accordion-content">Use Ember.js for its opinionated structure and extensive ecosystem, which simplify development and ensure long-term stability for web applications.</I.Content>
+    </A.Item>
+  </Accordion>
+
+  <style>
+    /* Bootstrap 5 Accordion styles (more or less) */
```

Comment:
you can also use a `<link` tag in here to load the actual bootsrap styles if you want to re-use the classes directly without re-defining everything in a `<style>` tag (I think toggle or switch or something does that as well)

Reply:
Good idea! Updated the example to `link` bootstrap styles